scripts/build/tauri-rename-ubuntu.sh

```diff
@@ -7,25 +7,23 @@ ls ../target/release ../target/release/bundle/deb
 # Used for release artifact
 # In release mode the name comes from tauri.conf.json
 # Using a glob for the source, there will only be one exe and one deb anyway
-cp ../target/release/firezone "$BINARY_DEST_PATH"-amd64
-cp ../target/release/firezone-gui-client.dwp "$BINARY_DEST_PATH"-amd64.dwp
-cp ../target/release/bundle/deb/*_amd64.deb "$BINARY_DEST_PATH"_amd64.deb
```

Comment:
I think the deb having an underscore where the other ones had a hyphen was just a convention, and Debian won't actually care if we break that, so this should be okay.